visivo/models/targets/sqlite_target.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Literal, Optional
+from visivo.models.base.base_model import BaseModel
+from visivo.models.targets.sqlalchemy_target import SqlalchemyTarget
+from pydantic import Field
+
+
+class Attachment(BaseModel):
+    schema_name: str = Field("Name of the schema to attach the target under.")
+    target: "SqliteTarget" = Field(
+        None,
+        description="Local SQLite database target to attach in the connection that will be available in the base SQL query.",
+    )
+
+
+class SqliteTarget(SqlalchemyTarget):
+    """
+    SqliteTargets hold the connection information to SQLite data sources.
+
+    !!! example {% raw %}
+
+        === "Simple"
+
+            ``` yaml
+            targets:
+              - name: sqlite_target
+                database: local/file/local.db
+                type: sqlite
+            ```
+
+        === "Additional Attached"
+            Attaching other SQLite databases allows you to join models between databases.
+
+            ``` yaml
+            targets:
+              - name: sqlite_target
+                database: local/file/local.db
+                type: sqlite
+                attach:
+                  - schema_name: static
```

Comment:
Not as it stands, feel like that is a pretty thin use case.  I exposed this because I needed it for the join model, but feel like it won't really be used much directly.

Reply:
I will leave that off this monster of a PR.